### DISCUSSION TITLE
新功能建议

### DISCUSSION THREAD

**Discussion by meinaOfficial** [immersive-translate/immersive-translate#1560] [Ideas]
### 新功能 | New Feature

1.翻译网站后同域名网站自动翻译；
2.点击扩展栏图标一键翻译网页；

### 为什么您想要添加这些功能 | Why you want to add these features

尤其是网页打开新连接还要麻烦点一遍翻译。
插件太多右击菜单容易点错。

**Comment by theowenyoung** [comment]
![CleanShot 2024-03-22 at 00 46 47@2x](https://github.com/immersive-translate/immersive-translate/assets/62473795/411eccb4-b723-4c6b-889d-dfca19987805)

选中这个即可。


![CleanShot 2024-03-22 at 00 47 17@2x](https://github.com/immersive-translate/immersive-translate/assets/62473795/8650a65d-ecea-4cb2-b9d2-23efd1d06b67)


点击这个即可一键翻译。
